<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationWindow</class>
 <widget class="QMainWindow" name="ApplicationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="HomePage">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ViewFamily">
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout_3"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="EditFamily">
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout_5"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLogout">
    <property name="title">
     <string>logout</string>
    </property>
   </widget>
   <addaction name="menuLogout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
